ts/data/case.ts

```diff
@@ -98,51 +103,49 @@ export interface Case {
   latestActivityDate: string;
 }
 
+export var caseFilters: FilterDefinition[] = [
```

Comment:
This is now ending up `undefined` when `sample.ts` attempts to access it. This is because of a circular dependency between `case.ts` and `sample.ts`. I think the best way to sort it out would be to create a new `table-components.ts` module that holds `caseFilters`. Other similar definitions that are shared between multiple table definitions can be moved here as well